pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/Codec.java

```diff
@@ -84,4 +87,23 @@
      * @throws IOException If it is impossible to read from the {@link ReadableSequentialData}
      */
     boolean fastEquals(@NonNull T item, @NonNull ReadableSequentialData input) throws IOException;
+
+    /**
+     * Converts a Record into a Bytes object
+     *
+     * @param item The input model data to convert into a Bytes object.
+     * @return The new Bytes object.
+     * @throws RuntimeException wrapping an IOException If it is impossible
+     * to write to the {@link WritableStreamingData}
+     */
+    default Bytes toBytes(@NonNull T item) {
+        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
```

Comment:
would be nice if used try with resources

Reply:
Chaned.